src/index.ts

```diff
@@ -458,7 +453,8 @@ function addSynctexCommands(app: JupyterLab, editorTracker: IEditorTracker, pdfT
       let widget = editorTracker.currentWidget;
       if (widget) {
         // Get the cursor position.
```

Comment:
I'd prefer a comment to exist explaining the synctex indexing vs. the editor's usual indexing.